### ISSUE TITLE
exit command does not work

### ISSUE THREAD

**Issue by Legion2** [AmyAssist/Amy#186]
the command `exit` should stop Amy but that does not work.
```
This is Amy. Copyright (c) 2018 the Amy project authors. For help run with flag -h.
22:50:45.586 [main] INFO  d.u.iaas.amyassist.amy.core.Core - run
22:50:47.051 [main] INFO  d.u.i.a.a.c.p.PluginLoader - loaded plugin Amy plugin alarmclock with 9 classes
22:50:47.095 [main] INFO  d.u.i.a.a.c.p.PluginLoader - loaded plugin Amy plugin calendar with 5 classes
22:50:47.105 [main] INFO  d.u.i.a.a.c.p.PluginLoader - loaded plugin Amy plugin email with 4 classes
22:50:47.113 [main] INFO  d.u.i.a.a.c.p.PluginLoader - loaded plugin Amy plugin example with 8 classes
22:50:47.132 [main] INFO  d.u.i.a.a.c.p.PluginLoader - loaded plugin Amy plugin spotify with 12 classes
22:50:47.138 [main] INFO  d.u.i.a.a.c.p.PluginLoader - loaded plugin Amy plugin systemtime with 3 classes
22:50:47.143 [main] INFO  d.u.i.a.a.c.p.PluginLoader - loaded plugin Amy plugin weather with 5 classes
22:50:47.158 [main] INFO  d.u.i.a.a.c.p.PluginLoader - loaded plugin Amy plugin navigation with 9 classes
22:50:48.528 [main] INFO  d.u.i.a.amy.httpserver.Server - start the server

amy> 22:50:50.890 [main] INFO  d.u.iaas.amyassist.amy.core.Core - running
22:50:51.639 [Thread-5] INFO  d.u.i.a.a.c.s.r.SpeechRecognizer - [INFORMATION] :: Speech Recognition activated
exit
22:51:04.435 [Thread-1] INFO  d.u.iaas.amyassist.amy.core.Core - stop
22:51:04.435 [Thread-1] INFO  d.u.i.a.amy.httpserver.Server - shutdown the server
22:51:04.457 [Thread-1] INFO  d.u.iaas.amyassist.amy.core.Core - stopped
```
after the `exit` command Amy is still running

**Comment by Legion2** [comment]
this bug must be fixed before #185 can be done

**Comment by Legion2** [comment]
https://github.com/AmyAssist/Amy/blob/dev/core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/speech/tts/TextToSpeech.java#L204 is never called
